### DISCUSSION TITLE
Could not load full workflow

### DISCUSSION THREAD

**Discussion by nicocopez** [h2020charisma/oranchada#47] [Q&A]
Hi,
I just  updated the version of Oranchada, and when trying to load the Workflow we saw during the Sofia hands on, i get the following error: 
Some of the links /nodes could not be reconstructed and were omitted from the workflow.
Missing node definitions:
  • orangecontrib.oranchada.widgets_pro.load_file.LoadFile
  • orangecontrib.oranchada.widgets_pro.xaxis_fine_calibration.XAxisFineCalibration
  • orangecontrib.oranchada.widgets_pro.load_file.LoadFile
  • orangecontrib.oranchada.widgets_pro.xaxis_fine_calibration.XAxisFineCalibration
  • orangecontrib.oranchada.widgets_pro.load_file.LoadFile
  • orangecontrib.oranchada.widgets_pro.xaxis_fine_calibration.XAxisFineCalibration
  • orangecontrib.oranchada.widgets_pro.load_file.LoadFile
  • orangecontrib.oranchada.widgets_pro.find_peaks.FindPeaks
  • orangecontrib.oranchada.widgets_pro.fit_peaks.Fit

Could this be due to the update?

Thank you for your time,

**Comment by nicocopez** [comment]
If I just open a new one, I can not see these widges.

**Comment by georgievgeorgi** [comment]
Could you please try to run oranchada by running this command in therminal?
```bash
python -m Orange.canvas -l4
```
In the terminal you will get error messages which may help to understand what is the problem. Please reply with the error messages.

**Reply by nicocopez** [reply]
If i translate to english, the error message is the following: 
Python was not found; run without arguments to install from Microsoft Store or disable this shortcut in Settings > Manage application execution aliases." This pop up when I wrote what you asked on after cd "C:\Program Files\Orange".
However, I can see Python on that folder, and if (once in that folder) I write "python", i get a windows thing saying the version.

**Reply by georgievgeorgi** [reply]
@kerberizer @vedina could you please help to run oranchada from the terminal in windows, or to tell how to check the error messages from the Orange GUI if possible at all?

**Reply by vedina** [reply]
Yes, there should be a log file in orange folders, Look for file charisma.log

**Comment by kerberizer** [comment]
> @kerberizer @vedina could you please help to run oranchada from the terminal in windows, or to tell how to check the error messages from the Orange GUI if possible at all?

Checking the error messages is easy: "View" -> "Log" from the menus should open the debug log panel.

**Reply by vedina** [reply]
Widgets not visible means there is an error when loading the add-onn

**Reply by vedina** [reply]
@nicocopez did you install Orange as portable (recommended), or via the installer?

Do you also have Spectroscopy add-on installed?

**Reply by nicocopez** [reply]
Sorry, the problem was me using the installed version instead of the portable. I uninstalled and deleted the intalable one now, so won't have this error again. I am back to the portable, and everything works.

Answering to the previous questions, for if the other issue is of any help: The add-on seems to be installed, Spectroscopy was installed (not on the portable version).

**Reply by vedina** [reply]
Ok, then I think we know what happened
